source/indepth/meteor/meteor.md.erb

```diff
@@ -33,5 +33,6 @@ Passenger can run Meteor apps both in non-packaged and packaged form. However, w
  * You do not have to install the Meteor runtime in the production environment.
  * Many Passenger production-grade features, such as multi-process management, [load balancing](<%= url_for "/indepth/request_load_balancing/index.html" %>), etc. only work properly if you run the application as multiple [processes](<%= url_for "/indepth/processes.html" %>). However, Passenger can only run your Meteor app with multiple processes if it is in packaged form. In non-packaged form, the various processes conflict with each other because they all try to start MongoDB.
  * Packaged apps use less resources because they do not have to load the Meteor development tools.
+ * It is also the [officially recommended](https://guide.meteor.com/deployment.html#deploying) way to deploy Meteor applications to production.
```

Comment:
I would emphasize how much it's recommended to bundle a Meteor app, with - __*strongly*__ word

Reply:
Yeah I'm not 100% sure about the classes either, but looks like they work just fine.

Reply:
I suggest moving the fact that it's the officially recommended way to the top of the list (and make it clear that this recommendation comes from Meteor itself, not just Phusion).